src/hub-rest-api/spec.yaml

```diff
@@ -844,7 +844,7 @@ paths:
       tags:
         - HubEvents
       operationId: ListEvents
-      summary: Fetch a page of Hub events
+      summary: Page of events
```

Comment:
Why not `Hub events`? "hub" seems like a more important word than "page" but feel free to argue otherwise

Reply:
HubEvents is already there in the title, and want to signify that it fetches multiple events